Masquerade4997CV/src/main/java/MasqCV/detectors/skystone/SkystoneDetectorv2.java

```diff
@@ -0,0 +1,243 @@
+package MasqCV.detectors.skystone;
+
+import com.disnodeteam.dogecv.detectors.DogeCVDetector;
+import com.disnodeteam.dogecv.filters.DogeCVColorFilter;
+import com.disnodeteam.dogecv.filters.GrayscaleFilter;
+import com.disnodeteam.dogecv.filters.LeviColorFilter;
+
+import org.opencv.core.Mat;
+import org.opencv.core.MatOfPoint;
+import org.opencv.core.Point;
+import org.opencv.core.Rect;
+import org.opencv.core.Scalar;
+import org.opencv.imgproc.Imgproc;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.LinkedList;
+
+import Library4997.MasqWrappers.DashBoard;
+
+public class SkystoneDetectorv2 extends DogeCVDetector {
```

Comment:
Delete this and move into SkystoneDetector.

Reply:
Okay